### PR TITLE
[MOVE] Format all source files to be consistent

### PR DESCRIPTION
**Pull request by CapCap**
### Description
 I formatted all the move and spec source files with the formatter built into Pontems IntelliJ plugin- many of the files were already formatted, some weren't

### Test Plan
whitespace only changes

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4526)

### CODE REVIEW

**Review by bowenyang007 — APPROVED** [review]

**Review by wrwg — CHANGES_REQUESTED** [review]
Wow, this is very cool! I missed the announcement from Pontem.

No way how we can land this for mainnet. This is too high of a risk. We would need to have a review of every single line of Move code changed here.

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 1]
```diff
@@ -545,19 +549,19 @@ module aptos_framework::account {
         alice
     }
 
-/*
-TODO bring back with generic rotation capability
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/account.move:549]: Let's just kill this TODO. Whoever wants to add this back in the future can go through the history and find it.

File: aptos-move/framework/aptos-framework/sources/code.move [hunk 2]
```diff
@@ -178,8 +178,8 @@ module aptos_framework::code {
         if (features::code_dependency_check_enabled())
             request_publish_with_allowed_deps(addr, module_names, allowed_deps, code, policy.policy)
         else
-            // The new `request_publish_with_allowed_deps` has not yet rolled out, so call downwards
-            // compatible code.
+        // The new `request_publish_with_allowed_deps` has not yet rolled out, so call downwards
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/code.move:181]: This looks bad. Can we throw {} around the if/else?

**Review by movekevin — APPROVED** [review]
Thanks for doing this Max! I've gone through every single line of change and all looks like harmless formatting changes.

**Comment by movekevin** [comment]
> Wow, this is very cool! I missed the announcement from Pontem.
> 
> No way how we can land this for mainnet. This is too high of a risk. We would need to have a review of every single line of Move code changed here.

I've spent 15 minutes going through every line and can certify everything is no-op changes.

File: aptos-move/framework/aptos-framework/sources/code.move [hunk 3]
```diff
@@ -195,10 +195,10 @@ module aptos_framework::code {
 
     /// Checks whether the given package is upgradable, and returns true if a compatibility check is needed.
     fun check_upgradability(
-            old_pack: &PackageMetadata, new_pack: &PackageMetadata, new_modules: &vector<String>) {
+        old_pack: &PackageMetadata, new_pack: &PackageMetadata, new_modules: &vector<String>) {
```
Comment (wrwg) [inline aptos-move/framework/aptos-framework/sources/code.move:198]: This is not a good formatting rule. The parameters should not be intended the same as the body

**Review by wrwg — COMMENTED** [review]
I'm still a bit skeptical, but since Kevin has looked in detail, go if you like.

**Pull request merged**